Dockerfile

```diff
@@ -1,12 +1,21 @@
-FROM python:3.11-slim-bookworm
+FROM python:3.11.0-slim-bullseye
 
-RUN apt update && apt install -y npm && \
-    npm i -g @bitwarden/cli && \
-    apt purge -y npm
+RUN apt-get update && \
+    apt-get upgrade -y && \
```

Comment:
I think `apt-get upgrade -y` is not needed as the base image gets updated regularly so there isn't usually anything to upgrade.

Reply:
I thought same as you, but I ran `docker scout` on the image and found one critical CVE with an outdated package on the base image. I can still remove it, but just wanted to let you know 👍🏾